tests/utils.py

```diff
@@ -84,6 +85,36 @@ def test_otx(self, mock_get=None, mock_post=None):
         ).start()
         self.assertEqual(report.get("success", False), True)
 
+    def test_InQuest_IOCdb(self, mock_get=None, mock_post=None):
+        report = inquest.InQuest(
+            "InQuest",
+            self.job_id,
+            self.observable_name,
+            self.observable_classification,
+            {"inquest_analysis": "iocdb_search"},
+        ).start()
+        self.assertEqual(report.get("success", False), True)
+
+    def test_InQuest_REPdb(self, mock_get=None, mock_post=None):
+        report = inquest.InQuest(
+            "InQuest",
```

Comment:
same as: https://github.com/intelowlproject/IntelOwl/pull/282/files#r540822823

Reply:
For all 3 analyzers, Right?. I'll follow #291 as reference